app/controllers/review_controller.rb

```diff
@@ -1,21 +1,36 @@
 class ReviewController < ApplicationController
   def index
     @card = current_user.card_for_review
+    respond_to do |format|
+      if @card.present?
+        format.html
+        format.json do
+          render json: {id: @card.id, image_url: @card.image.url(:medium), translated_text: @card.translated_text}
```

Comment:
Line is too long. [114/80]<br>Space inside { missing.<br>Space inside } missing.